crates/bevy_ui/src/render/mod.rs

```diff
@@ -344,9 +344,12 @@ pub fn prepare_uinodes(
             positions[3] + positions_diff[3].extend(0.),
         ];
 
+        let rect_size_v4 = uinode_rect.size().extend(0.).extend(0.);
+        let transformed_rect_size = (extracted_uinode.transform * rect_size_v4).xy().extend(1.);
```

Comment:
is the extend(1.) necessary? You don't use the z component after that so it seems like this doesn't do anything.

Reply:
Yes it was left over because rect_size is a vec3 I didn't notice, you are right it's not needed now.